src/Api/PersonalAccessTokens.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+* This file is part of the Gitlab API library.
+*
+* (c) Matt Humphrey <matth@windsor-telecom.co.uk>
+* (c) Graham Campbell <hello@gjcampbell.co.uk>
+*
+* For the full copyright and license information, please view the LICENSE
+* file that was distributed with this source code.
+*/
+
+namespace Gitlab\Api;
+
+use Symfony\Component\OptionsResolver\Options;
+
+class PersonalAccessTokens extends AbstractApi
+{
+    /**
+     * @param array $parameters {
+     *
+     *     @var string             $search            search text
+     *     @var string             $state             state of the token
+     *     @var int                $user_id           tokens belonging to the given user
+     *     @var bool               $revoked           whether the token is revoked or not
+     *     @var \DateTimeInterface $created_before    return tokens created before the given time (inclusive)
+     *     @var \DateTimeInterface $created_after     return tokens created after the given time (inclusive)
+     *     @var \DateTimeInterface $last_used_after   return tokens used before the given time (inclusive)
+     *     @var \DateTimeInterface $last_used_before  return tokens used after the given time (inclusive)
+     * }
+     *
+     * @return mixed
```

Comment:
Please move all return types to be real return types.

Reply:
Done.